installing/installing_ocp/install-central-config-options-ocp.adoc

```diff
@@ -1,13 +1,13 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="install-central-config-options-ocp"]
-= Optional - Configuring Central configuration options for RHACS using the Operator
+= Configuring Central configuration options for RHACS using the Operator
 :context: install-central-config-options-ocp
 include::modules/common-attributes.adoc[]
 
 toc::[]
 
 [role="_abstract"]
-This topic provides information about optional configuration options that you can configure using the Operator.
+You can configure optional settings for Central during installation when using the Operator.
```

Comment:
I like the change here, but it needs something to make it a but more clear. (Unless my ignorance of the subject is the problem.)

```suggestion
You can configure optional settings for your Central instance when installing by using the Operator.
```